src/main/java/io/jenkins/pluginhealth/scoring/model/Plugin.java

```diff
@@ -49,14 +55,19 @@ public class Plugin {
     @Column(name = "release_timestamp")
     private ZonedDateTime releaseTimestamp;
 
+    @Type(type = "json")
+    @Column(columnDefinition = "jsonb")
+    private Map<String, String> details;
+
     public Plugin() {
 
     }
 
-    public Plugin(String name, String scm, ZonedDateTime releaseTimestamp) {
+    public Plugin(String name, String scm, ZonedDateTime releaseTimestamp, Map<String, String> details) {
         this.name = name;
         this.releaseTimestamp = releaseTimestamp;
         this.scm = scm;
+        this.details = details;
```

Comment:
I wouldn't set the map in the constructor. You can probably mark the map as `final` and initialize it by default to an empty map.